packages/node/src/handlers.ts

```diff
@@ -280,6 +303,7 @@ export function parseRequest(
   }
 
   if (options.transaction && !event.transaction) {
+    // TODO: Use the real transaction here
```

Comment:
Is this a left over or is there something to be done here? What real transaction?!

Reply:
Removed